junction/base/admin.py

```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, unicode_literals
 
-# Third Party Stuff
```

Comment:
Does isort take care of ordering groups of library as well ? The reason for removing these comments I guess.

Reply:
It does, yes. It groups all third party library imports alphabetically.